build_container/docker_push_windows.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
Can this script merged into docker_push.sh?

Reply:
yeah i was wondering earlier if you would want to pull the common bits into a shared script, i can work on a refactor that parameterizes that script a bit

Reply:
@lizan tried it out, lmk what you think of the refactor